.gitignore

```diff
@@ -20,6 +20,7 @@
 .DS_Store
 *.pem
 /*.sql
+/*.patch
```

Comment:
I created some patch files to show the diff to the create_item and update_item sql functions, so I thought I'd update the gitignore here to ensure they don't land in the repo.